with-editor.el

```diff
@@ -518,7 +518,14 @@ which may or may not insert the text into the PROCESS' buffer."
                (if (file-name-absolute-p file)
                    (if (tramp-tramp-file-p default-directory)
                        (with-parsed-tramp-file-name default-directory nil
-                         (tramp-make-tramp-file-name method user host file hop))
```

Comment:
Wouldn't we be better off with `(concat (file-remote-p default-directory) file)`?